.github/dependabot.yml

```diff
@@ -0,0 +1,81 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
```

Comment:
`target-branch: "main"` isn't missing here?

Reply:
no, empty target-branch means deafult branch - in this case main